pages/book/create.js

```diff
@@ -0,0 +1,52 @@
+import Link from "next/link";
+import styled from "styled-components";
+import BookForm from "@/components/BookForm/BookForm";
+import Bottom from "@/components/Bottom/Bottom";
+
+export default function CreateBookPage({ onNewSubmit }) {
+  return (
+    <StyledMain>
+      <BookForm onNewSubmit={onNewSubmit} />
+      <Bottom>
+        <StyledLinkPlaceholder />
```

Comment:
if it's just a placeholder and serves no purpose yet, you should remove this component

Reply:
The Bottom component is a Layout component. I have to use the placeholder, to place the button.

Reply:
I kind of agree with Niklas, but I also see that you want to remember what you want to add. You could write a comment in this line with `//`

I'm trying to understand your `Bottom` component, is this going to be your navigation bar? Or is there more to it?

Reply:
Das ist die Navigationsleiste. Zur Zeit iste es eine Flex Box, werde diese aber später beim richtigen Styling noch in ein Grid verwandeln.

Reply:
This is the navigation bar. At the moment it is a flex box, but I will change it to a grid later when I style it correctly.

Reply:
Then I strongly recommend changing the name to `Navigation` or `NavigationBar` since `Bottom` is not very clear.